collections/_version2-english/9-fork/1-basic-knowledge.md

```diff
@@ -117,3 +126,15 @@ In this case, the threshold (`275,000` REP) is exceeded in Dispute Round 20. Tha
 
 ## When does a fork end?
 The period from the start to the end of a fork is referred to as the [forking period]({{glossary_forking_period}}). The forking period ends when either 60 days have passed, or more than 50% of [all theoretical REP]({{glossary_all_theoretical_rep}}) is migrated to one of the [child universes]({{glossary_child_universe}}). Whichever child universe receives the most migrated REP by the end of the forking period becomes the [winning universe]({{glossary_winning_universe}}), the other child universes become the [losing universe]({{glossary_losing_universe}}).
+
+## What happens to the parent universe?
+When a fork starts the [parent universe]({{glossary_parent_universe}}) becomes permanently locked. In the [locked universe]({{glossary_locked_universe}}), no new [markets]({{glossary_market}}) can be created and no markets can go through the [reporting]({{glossary_report}}) process. As a result of this, no markets can be [finalized](glossary_finalized_market). (See [Restrictions on use]({{url_restrictions_on_use}}) for details.)
+
+## What happens to the child universes?
+As mentioned [above](#when-does-a-fork-end), when the [forking period]({{glossary_forking_period}}) ends, [child universes]({{glossary_child_universe}}) become the [winning universe]({{glossary_winning_universe}}) or the [losing universe]({{glossary_losing_universe}}).
```

Comment:
```suggestion
As mentioned [above](#when-does-a-fork-end), when the [forking period]({{glossary_forking_period}}) ends, [child universes]({{glossary_child_universe}}) become either the [winning universe]({{glossary_winning_universe}}) or a [losing universe]({{glossary_losing_universe}}).
```

Reply:
@MicahZoltu 
Then I think
` become either the [winning universe]`
should be
`become either a [winning universe]`
Right?

Reply:
I think `the` is correct here since there is only one winning universe, but `a` is correct for losing since there are multiple losing universes.  Generally, you use `the` when talking about a singular thing and you use `a` when talking about a countable thing.

Reply:
👍　I got a step closer to the *real* smarty-pants.